src/main/java/core/basesyntax/TreeSetCharacters.java

```diff
@@ -17,7 +23,20 @@
  * Результат 2: acf</p>
  */
 public class TreeSetCharacters {
-    public String getUniqueCharacters(String fileName) {
-        return null;
+    public String getUniqueCharacters(String fileName) throws IOException {
+        Set<String> myTreeSet = new TreeSet<>();
+        File file = new File(fileName);
+        FileReader fr = new FileReader(file);
+        int data;
+        do {
+            data = fr.read();
+            myTreeSet.add(String.valueOf((char) data).toLowerCase());
```

Comment:
we can use checking `Character.isLetter(data)` if true: `myTreeSet.add(Character.toLowerCase(data));`